test/views/paye/PayeGovernmentSpendViewSpec.scala

```diff
@@ -26,13 +26,14 @@ import views.html.paye.PayeGovernmentSpendingView
 
 class PayeGovernmentSpendViewSpec extends TestConstants with ViewSpecBase {
 
-  implicit val request = PayeAuthenticatedRequest(testNino, FakeRequest("GET", "/annual-tax-summary/paye/treasury-spending"))
+  implicit val request =
+    PayeAuthenticatedRequest(testNino, FakeRequest("GET", "/annual-tax-summary/paye/treasury-spending"))
   lazy val payeGovernmentSpendingView: PayeGovernmentSpendingView = inject[PayeGovernmentSpendingView]
 
   "view" should {
     "have correct data and heading for given taxYear" in {
 
-      val view = payeGovernmentSpendingView(PayeAtsTestData.payeGovernmentSpendViewModel).body
+      val view = payeGovernmentSpendingView(PayeAtsTestData.payeGovernmentSpendViewModel, false).body
```

Comment:
extract these new false/true values to descriptive variables

Reply:
done